index.html

```diff
@@ -3158,7 +3158,10 @@ <h2 id="AudioDestinationNode">
           Giving a concrete example, if the audio hardware supports 8-channel
           output, then we may set <a href=
           "#widl-AudioNode-channelCount">channelCount</a> to 8, and render 8
-          channels of output.
+          channels of output. <span class="synchronous">An
+          <code>NotSupportedError</code> MUST be fired if the implementation
+          does not support changing the <a href=
+          "#widl-AudioNode-channelCount">channelCount</a></span>.
```

Comment:
Thanks for the explanation; I understand what Gecko is doing.

I'd rather not bake this condition into the spec. I think it would be ok for Gecko to return 2 even if the hardware has more channels (just like Chrome on Linux did until a couple of weeks ago).  And write it off as a bug in the implementation.  I think that's more useful from a user's viewpoint with fewer exceptional cases to consider.